internal/exec/stages/disks/luks.go

```diff
@@ -531,3 +570,149 @@ func (d LuksDump) hasFlag(flag string) bool {
 	}
 	return false
 }
+
+// collect the cex card domains xx.xxxx
+func getAllCexOnlineAPQNs() (string, error) {
+	apDir := "/sys/bus/ap/devices/"
+	uEvent := "/uevent"
+	var ret []string
+	// cards details are in form of link file points to card directory.
+	event, _ := os.ReadDir(apDir)
+	if len(event) == 0 {
+		return "", fmt.Errorf("adjunct processor not found")
+	}
+	// match the regexp for pattern xx.xxxx
+	for _, dir := range event {
+		match := cexRegx.FindStringSubmatch(dir.Name())
+		// if a match is found, read the uevent from that APQN directory
+		// to verify whether it corresponds to a CCA controller. This is
+		// intended for potential future use with an EP11 controller.
+		if match != nil {
+			filePath := apDir + strings.Join(match, "") + uEvent
```

Comment:
Should use `filepath.Join` here.

```suggestion
			filePath := filepath.Join(apDir, dir.Name(), "uEvent")
```